src/app/components/forms/checkbox-group/checkbox-group.component.ts

```diff
@@ -42,6 +42,13 @@ export class CheckboxGroupComponent implements ControlValueAccessor
     @Input()
     public checkboxValues:Array<{ caption:string, value:any }> = [];
 
+    /**
+     * @description en-/disable collapse functionality. If enabled, only the header is shown initially.
```

Comment:
```suggestion
     * @description If true, the checkbox-group gets collapsible. Default false.
```

Reply:
@johannabecker can you support us here? We need a good description for this input.